Back out "Revert D18299298: [pytorch][PR] Migrate conv3d from TH to ATen (CPU)"

ezyang opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#29286 Back out "Revert D18299298: [pytorch][PR] Migrate conv3d from TH to ATen (CPU)"**

Original commit changeset: 33057d5a91d1

Differential Revision: [D18349945](https://our.internmc.facebook.com/intern/diff/D18349945/)

ezyang committed:
Back out "Revert D18299298: [pytorch][PR] Migrate conv3d from TH to ATen (CPU)"

Original commit changeset: 33057d5a91d1

Differential Revision: [D18349945](https://our.internmc.facebook.com/intern/diff/D18349945/)

[ghstack-poisoned]

kostmo commented:
## CircleCI build failures summary
As of commit 7ac9c05:
* 0/1 recognized as flaky
* 1/1 failures introduced in this PR



Here are the [reasons each build failed](https://dr.pytorch.org/commit-details.html?sha1=7ac9c05fb84b0734f1513486dab61cd3467bd16a).

---
This comment was automatically generated by [Dr. CI](https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment).
Follow [this link to opt-out](https://dr.pytorch.org/admin/comments-opt-out.html) of these comments for your Pull Requests.


Please report bugs/suggestions on the [GitHub issue tracker](https://github.com/kostmo/circleci-failure-tracker/issues).

ezyang committed:
Update on "Back out "Revert D18299298: [pytorch][PR] Migrate conv3d from TH to ATen (CPU)""


Original commit changeset: 33057d5a91d1

Differential Revision: [D18349945](https://our.internmc.facebook.com/intern/diff/D18349945/)

[ghstack-poisoned]

ezyang committed:
Update on "Back out "Revert D18299298: [pytorch][PR] Migrate conv3d from TH to ATen (CPU)""


Original commit changeset: 33057d5a91d1

Differential Revision: [D18349945](https://our.internmc.facebook.com/intern/diff/D18349945/)

[ghstack-poisoned]

ezyang commented:
@houseroad could you restamp this? (IT's just a reland with correct bc testing)

houseroad reviewed (approved):
Looks good.
